content/fast/use-imagemin-to-compress-images/codelab-imagemin-grunt.md

```diff
@@ -15,11 +15,11 @@ glitch: imagemin-grunt
 This Glitch already contains `grunt` and `grunt-cli` for you, but you'll need to
 install `grunt-contrib-imagemin`.
 
-- Click the **Remix This** button to make the project editable.
+- Click the **Remix to Edit** button to make the project editable.
 
 <web-screenshot type="remix"></web-screenshot>
 
-- Click the **Status** button.
+- Click the **Tools** button.
 
 <web-screenshot type="status"></web-codelab>
```

Comment:
Screenshot needs to be updated

Reply:
I think we should remove the `web-screenshot` elements. Glitch changes their UI so frequently it's too hard to keep them in sync. Are you cool deleting them in this PR?

Reply:
Yeah for sure - I can go through and remove them where they show up.

Reply:
Just to clarify - do you want me to only remove the tools screenshots or just all web-screenshot elements?

Reply:
ah sorry about that. all web-screenshot elements should be removed.